main.go

```diff
@@ -41,6 +47,7 @@ func runMain() error {
 	log.SetFlags(0) // Removes default timestamp flag
 
 	args := procArgs()
+	log.Printf("Running version: %s", version)
```

Comment:
Nit: all the other log messages start with lowercase (pls fix for the recorder too)